CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [8.8.x] - 2025-01-??
 
+- Added: Preset option to place all majors or pickups logically.
```

Comment:
we'll need to mention that it's an experimental setting, and possibly also what the point of this setting is